README.md

```diff
@@ -1,5 +1,7 @@
 # Xendit Java Library
 
+[ ![Download](https://api.bintray.com/packages/xendit/android/xendit-java-lib/images/download.svg?) ](https://bintray.com/xendit/android/xendit-java-lib/latest/link)
```

Comment:
Can we make the text clearer like `Download from JCenter`?

Reply:
Sure

Reply:
Turn out this will only change the `alt` tag. The visual `Download` text is from bintray's svg (so shitty lol)